practice/src/index.html

```diff
@@ -7,10 +7,60 @@
         <title>Todo List app - TypeScript</title>
         <link rel="stylesheet" href="./styles/index.css">
         <link rel="shortcut icon" href="./assets/favicon.8602d591.ico" type="image/x-icon">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" />
     </head>
     <body>
-        
+        <main class="app">
+            <header class="app-heading">
+                <h1 class="heading-title">Todo list with TypeScript</h1>
+                <p class="heading-description">
+                    Build from <a href="https://www.typescriptlang.org/" class="underline-text" target="_blank">TypeScript</a>
+                    and <a href="https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller" class="underline-text" target="_blank">MVC model</a>
+                </p>
+            </header>
+            <div class="app-todo-content">
+                <div class="content-input">
+                    <i class="fa fa-check check-all"></i>
+                    <input id="task-input" class="task-input" type="text" placeholder="What will you focus on today?">
+                </div>
+                <ul class="list-of-tasks">
+                    <!-- Todo list task item will be added here  -->
+                </ul>
+                <div class="content-action">
+                    <p class="total-item"></p>
+                    <ul class="task-filter">
+                        <li class="task-filter-item"><a data-action="all" href="#all" class="selected item all active-btn" tabindex="0">All</a></li>
+                        <li class="task-filter-item"><a data-action="active" href="#active" class="active item selected" tabindex="0">Active</a></li>
+                        <li class="task-filter-item"><a data-action="completed" href="#completed" class="completed item selected" tabindex="0">Completed</a></li>
+                    </ul>
+                    <p class="clear-completed">Clear completed</p>
```

Comment:
do you have handle clear completed feature?

Reply:
Yes, It is a feature of the app